packages/story-editor/src/components/panels/document/taxonomies/taxonomies.js

```diff
@@ -18,18 +18,146 @@
  * External dependencies
  */
 import { __ } from '@web-stories-wp/i18n';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  BUTTON_VARIANTS,
+  DropDown,
+  Headline,
+  Input,
+  Text,
+  themeHelpers,
+  THEME_CONSTANTS,
+} from '@web-stories-wp/design-system';
+import { useCallback, useMemo, useState } from '@web-stories-wp/react';
+import { v4 as uuidv4 } from 'uuid';
 /**
  * Internal dependencies
  */
+import styled from 'styled-components';
 import { SimplePanel } from '../../panel';
+import { HierarchicalInput } from '../../../form';
+import { noop } from '../../../../utils/noop';
+
+const ContentArea = styled.div`
+  label,
+  * > label {
+    ${({ theme }) =>
+      themeHelpers.expandPresetStyles({
+        preset:
+          theme.typography.presets.label[
+            THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL
+          ],
+        theme,
+      })};
```

Comment:
The labels that are a part of the `Input` component have a `primary` color and a font size that is bigger than what's specified in the designs. This catches all labels in this section and makes them pretty.